src/items.cpp

```diff
@@ -4707,57 +4665,67 @@ bool itemSpriteIsBreastpiece(int sprite)
 	return false;
 }
 
-real_t getArtifactWeaponEffectChance(ItemType type, Stat& wielder, real_t* effectAmount)
+real_t getArtifactWeaponEffectChance(const ItemType type, Stat& wielder, real_t* const effectAmount)
 {
 	if ( type == ARTIFACT_AXE )
 	{
-		real_t percent = 25 * (wielder.PROFICIENCIES[PRO_AXE]) / 100.f; //0-25%
+		const real_t percent = 25 * (wielder.PROFICIENCIES[PRO_AXE]) / 100.f; //0-25%
```

Comment:
adding const to variables that aren't function parameters isn't necessary

Reply:
This will be kept in mind for future commits, except in cases where it is important (which I don't foresee coming up often). If you prefer I undo the changes in this commit, let me know, otherwise, going forward.